bears/configfiles/DockerfileLintBear.py

```diff
@@ -29,8 +29,16 @@ class DockerfileLintBear:
         'info': RESULT_SEVERITY.INFO}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return '--json', '-f', filename
+    def create_arguments(filename, file, config_file,
+                         rule_file: str=''):
+        """
+        :param rule_file:
+            A yaml rule file for `dockerfile_lint`.
```

Comment:
The setting name should include `dockerfile_lint` (or somehow the toolname), so it's not ambiguous with other bear settings implementing a similar functionality. Maybe `dockerfile_lint_config` or `dockerfile_lint_rule_file`. Those setting names can be specific because this is just a convenience feature for compatibility, if people are too lazy to migrate to coala's "native" settings.